### PR TITLE
Started cohort type guide - first draft

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

File: docs/Data_Portal/Users_Guide/cohort_builder.md [hunk 1]
```diff
@@ -90,3 +90,41 @@ Users can then perform the following actions:
...
 * Analyze data from the cohort in the [Analysis Center](analysis_center.md)
+
+
+## Cohort Types
+
+Depending on how they are modified or created, cohorts can have different types of filters and thus behave differently:
+* __General Queries__
+* __Specific List of Cases__
+
+### General Queries
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/cohort_builder.md:101]: not loving calling them 'general queries'. any thoughts, @wwysoc2 ?

**Comment by wteouchicago** [comment]
@wwysoc2 - I made some changes. lemme know what you think.

File: docs/Data_Portal/Users_Guide/cohort_builder.md (hunk 1 above)
Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/cohort_builder.md:101]: Maybe "Custom Queries"? I think it gives the sense that users are always in control here, which they are.

Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/cohort_builder.md:101]: Also, add a return before the bullets.

Comment (wteouchicago) [reply docs/Data_Portal/Users_Guide/cohort_builder.md]: all done. lemme know if it's ok to merge. @wwysoc2

**Comment by wwysoc2** [comment]
Changes look good to me.  Let's run them by @bullenc to make sure they make sense.

File: docs/Data_Portal/Users_Guide/cohort_builder.md [hunk 2]
```diff
@@ -90,3 +90,40 @@ Users can then perform the following actions:
 
 * Download files associated with the cohort from the [Repository](Repository.md)
 * Analyze data from the cohort in the [Analysis Center](analysis_center.md)
+
+## Cohort Types
+
+Depending on how they are modified or created, cohorts can have different types of filters and thus behave differently:
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/cohort_builder.md:96]: is the behavior difference mainly with regard to changes in data releases that may impact the cases returned or not returned? Perhaps a qualifier at the end should add: "... and thus behave differently with regards to which cases are included in the cohort"?

File: docs/Data_Portal/Users_Guide/cohort_builder.md [hunk 3]
```diff
@@ -90,3 +90,40 @@ Users can then perform the following actions:
...
+
+* __Custom Queries__
+* __Specific List of Cases__
+
+### Custom Queries
+
+Examples of cohorts with custom queries would be:
```
Comment (bullenc) [inline docs/Data_Portal/Users_Guide/cohort_builder.md:103]: Maybe add some initial context to what is meant by a custom query cohort; if I understand correctly, maybe something like, "One way cohorts can be built is by customizing filters and queries in the Cohort Builder Tool or in the Query Expressions section. Examples of cohorts... "

File: docs/Data_Portal/Users_Guide/cohort_builder.md (hunk 2 above)
Comment (wteouchicago) [reply docs/Data_Portal/Users_Guide/cohort_builder.md:96]: yup! that's the main difference in behaviour! I'll edit.

File: docs/Data_Portal/Users_Guide/cohort_builder.md (hunk 3 above)
Comment (wteouchicago) [reply docs/Data_Portal/Users_Guide/cohort_builder.md:103]: will do 👍

**Comment by wteouchicago** [comment]
updated. let me know if it's any better. @bullenc / @wwysoc2  🙇‍♀️

**Review by wteouchicago — APPROVED** [review]
Bill has approved this.

**Pull request merged**